app/views/api/v2/split_registries/show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.splits do
+  @split_registry.splits.each do |split|
+    json.set! split.name do
+      json.weights split.registry
+      json.feature_gate split.feature_gate?
+    end
+  end
+  json.merge!({})
```

Comment:
This seems a little ugly but it was the only way I found to force the `splits` key to appear with an empty object as a value. If the loop is empty, otherwise the split key disappears entirely. Is there a more revealing approach?